app.yaml

```diff
@@ -1,7 +1,8 @@
 runtime: python37
+instance_class: F4
 automatic_scaling:
   min_instances: 1
```

Comment:
(Not related to these changes just a question trying to understand app engine stuff) Why do we have the min_instances as 1 instead of 0? (https://cloud.google.com/appengine/docs/standard/python3/config/appref#scaling_elements <- mentioned that you are charged by number of instances whether receiving traffic or not so if min is 1 will always cost at least 1 vs if min were 0 would not charge if no traffic?)

Reply:
Spinning up a new instance is fairly quick but isn't free; it does take some time. It does incur cost, but having at least one instance ready reduces latency for the first request after being idle for a while.

I expect this to be used pretty infrequently, in which case most requests would be running against new instances so would always incur the extra latency. Of course, that means that we're paying for an instance that is going mostly unused which is not ideal. However, right now, reducing latency is critical.